src/gatsby-ssr.js

```diff
@@ -12,7 +12,9 @@ exports.onRenderBody = ({ setPostBodyComponents }, pluginOptions) => {
     src: pluginOptions.srcUrl,
     "data-website-id": pluginOptions.websiteId,
     "data-auto-track": 'autoTrack' in pluginOptions ? pluginOptions.autoTrack : true,
-    "data-do-not-track": 'respectDoNotTrack' in pluginOptions ? pluginOptions.respectDoNotTrack : true
+    "data-do-not-track": 'respectDoNotTrack' in pluginOptions ? pluginOptions.respectDoNotTrack : true,
+    "data-cache": 'dataCache' in pluginOptions ? pluginOptions.dataCache : false,
+    "data-domains": 'dataDomains' in pluginOptions ? pluginOptions.dataDomains : ''
```

Comment:
I'm not sure, how Umami will handle an empty string, since the property is still present.

Reply:
I was also thinking about this. I think if it is an empty string it will be on the script tag like `data-domains=""` Maybe  `undefined` is better, so that react discards the attribute.